migrations/Version20220824200533.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20220824200533 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE User DROP name, DROP surname, DROP adress, DROP house_number, DROP post_number, DROP city, DROP state');
```

Comment:
@marija-mikic zasto mices stupce u dizanju migracije?

Reply:
tu sam brisala jedan stupac koji sam greškom ukucala

Reply:
da, al tu ih brises vise od jednog

Reply:
iskreno, nisam ga ja ručno ukucavala  i ne znam zašto je tu DROP..tako mi je i na migraciji 342

Reply:
molim te pogledaj sto radis u migracijama i razmisli sto ti treba, a sto ne :)

Reply:
dali to znači da sve u migracijama gdje mi je drop brišem?

Reply:
dropas samo stupce koji ti ne trebaju, a ne sve

Reply:
migracija ti puca jer dropas stupac kojeg nema